actions/java-gradle-build-jib-image/action.yaml

```diff
@@ -0,0 +1,53 @@
+name: "Build jib image"
+description: "Build jib image using gradle"
+
+inputs:
+  image-artifact-name:
+    descritption: "Artifact name to upload jib docker image (see https://github.com/actions/upload-artifact and https://github.com/actions/download-artifact)"
+    required: false
+    default: "docker-image"
```

Comment:
I think we should enforce the artifact name. I don't think any developer wants his artifact to be called `docker-image`.

Reply:
This is the name for the CI artifact that contains the image from a previous job and has nothing to do with the image tag for publishing